src/base/bittorrent/private/filterparserthread.cpp

```diff
@@ -134,6 +135,12 @@ int FilterParserThread::parseDATFilterFile()
     int start = 0;
     int endOfLine = -1;
     int nbLine = 0;
+    int parseErrorCount = 0;
+    const auto addLog = [&parseErrorCount](const QString &msg)
+    {
+        if (parseErrorCount < MAX_LOG_LINES)
```

Comment:
Is it expected behavior you have `MAX_LOG_LINES - 1` errors in log (plus one line with info about extra error count)?